README.yaml

```diff
@@ -58,6 +58,8 @@ description: |-
 
 # How to use this project
 usage: |-
+  ### Based on VPC id
```

Comment:
Add this after `usage: |-`

**IMPORTANT:** Using the `master` branch is just an example. Do not pin to `master` in your code because there may be breaking changes between releases. Instead pin to the release tag (e.g. `?ref=tags/x.y.z`) of one of our [latest releases](https://github.com/cloudposse/terraform-aws-vpc-peering/releases).